### DISCUSSION TITLE
feat: allow individual field retrieval from Org Browser

### DISCUSSION THREAD

**Discussion by randi274** [forcedotcom/salesforcedx-vscode#3938] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to retrieve individual fields from a given object using the Org Browser tab.

**Describe the solution you'd like**
I'd like to see an individual download option like we use from our other great folder-supported metadata, like Email Templates: 
![Screen Shot 2022-03-16 at 4 35 00 PM](https://user-images.githubusercontent.com/9795193/158691047-6ccb43bc-668f-44cb-a234-5056fe62e41f.png)

**Describe alternatives you've considered**
Right now, I have to download the entire object to get the related fields, which is not always a convenient solution if my setup flow uses pre- or post-deploy. Alternatively, I can use the CLI commands to accomplish this (sfdx force:source:retrieve -m CustomField:Opportunity.TrackingNumber__c) , but I would prefer to use the Org Browser. 😃

**Additional context**
Here's the limited options I have for custom objects now: 
![Screen Shot 2022-03-16 at 4 34 15 PM](https://user-images.githubusercontent.com/9795193/158691185-1679b540-48aa-4a0d-bd02-f5ec50dd5cc9.png)

**Comment by jefersonchaves** [comment]
Yes, please, often I go to CLI just because this feature is missing on VsCode.
Not way too often but it would be nice.

**Comment by mshanemc** [comment]
done!
